earth2studio/lexicon/ifs.py

```diff
@@ -47,6 +47,17 @@ def build_vocab() -> dict[str, str]:
             "msl": "msl::sfc::",
             "tcwv": "tcwv::sfc::",
             "tp": "tp::sfc::",
+            "skt": "skt::sfc::",
+            "slor": "slor::sfc::",
+            "sdor": "sdor::sfc::",
+            "lsm": "lsm::sfc::",  # Should I remove this? Helps with debugging but not sure if it's needed
```

Comment:
If this is supported by IFS then no harm in leaving it.

Reply:
If its not added to the base lexicon then adding an entry for `"lsm: Landsea mask" wouldnt hurt